gpt_index/embeddings/openai.py

```diff
@@ -107,30 +107,41 @@ class OpenAIEmbedding(BaseEmbedding):
             - "babbage"
             - "ada"
             - "text-embedding-ada-002"
+        
+        deployment_name (str): Deployment of model for AzureOpenAI. Defaults to None.
```

Comment:
type as Optional[str]

Reply:
also this is more general than just AzureOpenAI right? would make that clear in the docstring (e.g. give Azure OpenAI deployment name as an example, rather than saying this is specifically for AzureOpenAI).

Reply:
As far as I know, deployment is only used for AzureOpenAI. You can find "Deployment operations are only available for the Azure API type." in https://github.com/openai/openai-python/blob/main/openai/api_resources/deployment.py

Reply:
oh interesting, i didn't know that. ok